pkg/anypoint/client_test.go

```diff
@@ -162,10 +165,11 @@ func TestClient(t *testing.T) {
 	logrus.Info(i, contentType)
 	assert.NotEmpty(t, i)
 	assert.Empty(t, contentType)
-	// fileContent, err := client.GetExchangeFileContent("https://abc.com", "zip", "mainfile")
-	// assert.Nil(t, err)
-	// assert.NotEmpty(t, fileContent)
 	events, err := client.GetAnalyticsWindow()
 	assert.Nil(t, err)
 	assert.Equal(t, 1, len(events))
+
+	go client.auth.Stop()
```

Comment:
The auth has a channel, and it wasn't being mocked, causing a data race warning in the tests.